src/quo2/core.cljs

```diff
@@ -307,8 +307,8 @@
 (def audio-tag quo2.components.tags.context-tag.view/audio-tag)
 (def community-tag quo2.components.tags.context-tag.view/community-tag)
 
-;;;; Title
-(def title quo2.components.text-combinations.title.view/title)
+;;;; Text combinations
+(def text-combinations quo2.components.text-combinations.view/view)
```

Comment:
Based on our Quo guidelines , the correct directory structure is `components/text_combinations/text_combinations/` and the public var in `quo2.core` should probably be named `text-combinations-text-combinations`.

But... our convention here looks weird with the repeated name. It would be good to talk to a designer to rename the the component `Text combinations` to something else, to avoid the case of having `quo/text-combinations-text-combinations`. They're open to such changes, and it's easy for them.

@J-Son89, what's your opinion about this sort of naming weirdness?

Reply:
Yep, for me it's better the consistency but best for sure is if we ask designers to make the changes. Afaik it's no so painful for them to do so 👌